doc/functions/cva.rst

```diff
@@ -0,0 +1,172 @@
+``cva``
+=======
+
+`CVA (Class Variant Authority)`_ is a concept from the JavaScript world and used
+by the well-known `shadcn/ui`_.
+CVA allows you to display a component with different variants (color, size, etc.),
+to create highly reusable and customizable templates. This function work best by being combined with
+`TwigComponent`_
+
+.. note::
+
+    The ``cva`` function is part of the ``HtmlExtension`` which is not
+    installed by default. Install it first:
+
+    .. code-block:: bash
+
+        $ composer require twig/html-extra
+
+    Then, on Symfony projects, install the ``twig/extra-bundle``:
```

Comment:
Similar notes are currently located at the end of the doc pages

* https://twig.symfony.com/doc/3.x/filters/markdown_to_html.html
* https://twig.symfony.com/doc/3.x/functions/timezone_names.html
* https://twig.symfony.com/doc/3.x/functions/html_classes.html

Reply:
👀